packages/ui/src/components/Tabs/tabs.tsx

```diff
@@ -23,6 +23,7 @@ export type TabsProps = Pick<DefaultTabsProps,
 'activeKey' |
 'destroyInactiveTabPane' |
 'onChange' |
+'tabBarExtraContent' |
```

Comment:
What's this used for? Can we solve problem with a flex `div` / `Space`, since tabs can be vertical, and this prop is not used by our design team